media/js/base/core-datalayer-init.js

```diff
@@ -48,10 +48,17 @@
                 var validParams = analytics.getAMOExperiment(params);
 
                 if (validParams) {
+                    // UA
                     dataLayer.push({
                         'data-ex-name': validParams['experiment'],
                         'data-ex-variant': validParams['variation']
                     });
+                    // GA4
+                    dataLayer.push({
+                        event: 'experiment_view',
```

Comment:
Seems like people arriving with these parameters have already viewed the experiment. Maybe this event would be better named `experiment_enroll`?